velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -328,6 +328,25 @@ std::unique_ptr<ParquetTypeWithId> ReaderBase::getParquetColumnInfo(
           // the "repeated info" which we are ignoring here, hence we set the
           // isRepeated to true eg
           // https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#lists
+          if (schemaElement.converted_type == thrift::ConvertedType::LIST &&
```

Comment:
need to move comments above to line 350. also better to have comments to explain added piece of code.